src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -200,7 +204,8 @@ protected Task Stop()
                         SignatureHelpProvider = new SignatureHelpOptions
                         {
                             TriggerCharacters = new string[] { " " } // TODO: Other characters here?
-                        }
+                        },
+                        DocumentFormattingProvider = false
```

Comment:
Should this be `true`?  Or are you doing this as a workaround to the FormattingOptions issue?